src/test/resources/example1.json

```diff
@@ -0,0 +1,9 @@
+{
+  "id": "0192",
+  "case_ref": "1537879748168579",
+  "jurisdiction": "SSCS",
+  "classification": "NEW_APPLICATION",
```

Comment:
isn't `zipFileName` required?

Reply:
not my change ... it was actually merged into master today.

Reply:
sure ... ill change this.